_field-types/supported-field-types/knn-vector.md

```diff
@@ -267,6 +267,238 @@ return Byte(bval)
 ```
 {% include copy.html %}
 
+## Faiss byte vector
```

Comment:
We can remove entire content here and just use whatever we have for lucene byte vector. 
The content in here can be moved to tech blog if we want.

Reply:
As lucene does not support IVF, we can change the lucene example with faiss example including new IVF index example.